central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,96 @@
+package scheduler
+
+import (
+	"bytes"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/stackrox/rox/pkg/branding"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+var expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `
```

Comment:
If strings don't change, please make them `const`-s. If there are multiple top-level var-s or const-s, you can unify them in a block.

```go
const (
	expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `...
	`

	expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders = `...
	`

	// and so on
)
```

Reply:
Okay, done.